tests/src/test/scala-2.13+/cats/tests/LazyListSuite.scala

```diff
@@ -52,6 +55,16 @@ class LazyListSuite extends CatsSuite {
     assert(LazyList.empty[Int].show === (s"LazyList()"))
   }
 
+  test("Avoid all evaluation of LazyList#foldRightDefer") {
```

Comment:
Just to confirm: this test hangs without the change above?

Reply:
Yes,  The test never finishes.